File: official/docs/node/current/referral-customers/add-payment-method-with-credit-card.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  const paymentMethod = await client.BetaReferralCustomer.addPaymentMethod('cus_...', 'card_...');

  console.log(paymentMethod);
})();

File: official/docs/python/current/referral-customers/add-payment-method-with-credit-card.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

payment_method = client.beta_referral_customer.add_payment_method(
    stripe_customer_id="cus_...",
    payment_method_reference="card_...",
    primary_or_secondary="primary",
)

print(payment_method)
